config/client.lua

```diff
@@ -0,0 +1,43 @@
+return {
+    useTarget = false,
+    zones = {
+        sandyOccasions = {
+            businessName = "Vehicle Sales Contract - Larry's Vehicle Sales",
+            sellVehicle = vec4(1235.61, 2733.44, 37.4, 0.42),
+            buyVehicle = vec4(1213.31, 2735.4, 38.27, 182.5),
+            points = {
+                vec3(1338.3748779297, 2645.0153808594, 36.0),
+                vec3(1098.9381103516, 2621.7487792969, 36.0),
+                vec3(1117.9478759766, 2822.0729980469, 36.0),
+                vec3(1370.98828125, 2859.197265625, 36.0)
+            },
+            spots = {
```

Comment:
these names are unclear. What are points? spots? These just sound like synonyms of coordinates. Consider renaming to be more descriptive and/or add comments to describe what these fields mean

Reply:
Well points would be the collection of points that form the zone. Spots are just the vehicle spots in which the for sale cars get placed. I can rename I guess.